test/CMakeLists.txt

```diff
@@ -4,13 +4,10 @@ include_directories(../third_party/sqlite/include)
 include_directories(include)
 
 add_subdirectory(api)
-add_subdirectory(appender)
-add_subdirectory(common)
-add_subdirectory(extension)
+# add_subdirectory(appender) add_subdirectory(common)
```

Comment:
Was this intentional?!

Reply:
No I already rollback this change. It's for speed up the local testing 😄

Reply:
You can run individual tests using `build/debug/test/unittest "Basic test of C API"`, or run all the capi tests using `build/debug/test/unittest "[capi]"`. No need to disable the compilation of the other tests :)

Reply:
OK, thanks!